subiquity/core.py

```diff
@@ -98,5 +111,12 @@ def unhandled_input(self, key):
         if key == 'f1':
             if not self.showing_help:
                 self.ui.right_icon.open_pop_up()
+        elif key == 'ctrl z':
+            self.debug_shell()
         else:
             super().unhandled_input(key)
+
+    def debug_shell(self):
+        self.run_command_in_foreground(
+            "clear && echo {} && bash".format(shlex.quote(DEBUG_SHELL_INTRO)),
```

Comment:
```
-            "clear && echo {} && bash".format(shlex.quote(DEBUG_SHELL_INTRO)),
+            "clear && read -p {} start && clear && bash".format(shlex.quote(DEBUG_SHELL_INTRO)),
```

This way, we wait for user to read text and hit enter.
Then clear the screen, then show the prompt.

On 80x24 this is a lot of intro text, that we would want to clear out.

Reply:
(also fine with confirm and not clearing intro text)

Reply:
I'm not completely sure I agree with this... it's no larger an amount of text than the default motd is it?

Reply:
@mwhudson 

Ok about the length of text.

But what about Enter to confirm the text first, prior to getting shell? Useful, or not really? (the `&& read -p` portion)